### PR TITLE
Formalize imbalance fees

### PR DESCRIPTION
**Pull request by palango**

### CODE REVIEW

File: mediation_fees.rst [hunk 1]
```diff
@@ -72,21 +84,28 @@ From (1) follows:
...
 This leads to
 
 .. math::
 
-    {fee}_{out} = c - b = c - \frac{c - f - i}{1+q}
+    {fee}_{out} = c - b = c - \frac{c - f - i(b)}{1+q}
 
+Here one can see that the calculation depends on both :math:`b` and :math:`c`. This formula doesn't have a symbolic solution for arbitrary functions :math:`i(x)`.
+
+We approximate the solution by calculating :math:`b' = \frac{c - f}{1+q}` and than use that to solve for :math:`b` (which is the first iteration towards the solution):
```
Comment (karlb) [inline mediation_fees.rst]: Maybe add 
```suggestion
We approximate the solution by calculating :math:`b \approx b' = \frac{c - f}{1+q}` and than use that to solve for :math:`b` (which is the first iteration towards the solution which assumes :math:`i = 0`):
```

File: mediation_fees.rst [hunk 2]
```diff
@@ -72,21 +84,28 @@ From (1) follows:
...
+
+.. math::
+
+    {fee}_{out} \approx c - b = c - \frac{c - f - i(b')}{1+q}
```
Comment (karlb) [inline mediation_fees.rst]:
```suggestion
    {fee}_{out} = c - b \approx c - \frac{c - f - i(b')}{1+q}
```

File: mediation_fees.rst [hunk 3]
```diff
@@ -97,13 +116,19 @@ From (2) follows:
...
 .. math::
 
-    {fee}_{in} = a - c = \frac{c + f + i}{1-q} - c
+    {fee}_{in} = a - c = \frac{c + f + i(a)}{1-q} - c
+
+Here the same approximation approach is used for the imbalance fee. The approximation :math:`i(a')` with :math:`a' = \frac{c + f}{1+q}` is used in the symbolic solution.
+
+.. math::
+
+    {fee}_{in} \approx a - c = \frac{c + f + i(a')}{1-q} - c
```
Comment (karlb) [inline mediation_fees.rst]:
```suggestion
    {fee}_{in} = a - c \approx \frac{c + f + i(a')}{1-q} - c
```

File: mediation_fees.rst (hunk 2 above)
Comment (palango) [reply mediation_fees.rst]: That's correct! 👀

**Pull request merged**